src/hexo/themes/sonarwhal/layout/partials/scan-result.hbs

```diff
@@ -38,6 +38,10 @@
                             <p class="scan-overview__subheader">version</p>
                             <p class="scan-overview__body--purple">{{'or' version '-'}}</p>
                         </div>
+                        <div class="scan-overview--config">
+                            <p class="scan-overview__subheader">Scan Configuration</p>
+                            <a class="scan-overview__body__link" href="#">View JSON file</a>
```

Comment:
I've [updated](https://github.com/sonarwhal/sonarwhal.com/commit/9d1a4a88174334e3e270d86085eb2fa407c1848e) this PR's related commit to have the link pointing to `config/{{id}}`.

See also: https://github.com/sonarwhal/sonarwhal.com/pull/324.